rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -252,17 +277,29 @@ public boolean checkJobPermissionMethod(String sessionId, String method, String
 
     @Override
     public RestMapPage<Long, ArrayList<UserJobData>> revisionAndJobsInfo(String sessionId, int index, int limit,
-            boolean myJobs, boolean pending, boolean running, boolean finished) throws RestException {
+            boolean myJobs, boolean pending, boolean running, boolean finished, String sortParams)
+            throws RestException {
         try {
             Scheduler s = checkAccess(sessionId, "revisionjobsinfo?index=" + index + "&limit=" + limit);
             String user = sessionStore.get(sessionId).getUserName();
 
             boolean onlyUserJobs = (myJobs && user != null && user.trim().length() > 0);
+            List<SortParameter<JobSortParameter>> sortParameterList;
+            if (Strings.isNullOrEmpty(sortParams)) {
+                sortParameterList = DEFAULT_JOB_SORT_PARAMS;
+            } else {
+                try {
+                    sortParameterList = createJobSortParams(sortParams);
+                } catch (Exception e) {
+                    logger.warn("Invalid sort parameter string, using default : " + sortParams);
+                    sortParameterList = DEFAULT_JOB_SORT_PARAMS;
+                }
+            }
 
             Page<JobInfo> page = s.getJobs(index,
                                            limit,
                                            new JobFilterCriteria(onlyUserJobs, pending, running, finished),
-                                           DEFAULT_JOB_SORT_PARAMS);
+                                           sortParameterList);
```

Comment:
sortParameterList -> orderedParameterList

Reply:
orderedParameterList ? I don't understand. It is one parameter containing the list of sort directives.

Reply:
the type is
List<SortParameter<JobSortParameter>>

Reply:
OK